tools/wpt/run.py

```diff
@@ -686,6 +686,22 @@ def setup_kwargs(self, kwargs):
         pass
 
 
+class WebKitTestRunner(BrowserSetup):
+    name = "wktr"
+    browser_cls = browser.WebKitTestRunner
+
+    def install(self, channel=None):
+        raise NotImplementedError
+
+    def setup_kwargs(self, kwargs):
+        if kwargs["binary"] is None:
+            binary = self.browser.find_binary(self.venv.path, None)
+
+            if binary is None:
+                raise WptrunError("Unable to find servo binary in PATH")
```

Comment:
Servo?

Reply:
Gah. There's not really any point in having that function at all. Removed.